store/entity-store.js

```diff
@@ -236,7 +236,7 @@ window.D2L.Siren.EntityStore = {
 		const expandEntities = [];
 		const entities = [];
 		expandEntities.push(entity);
-		entityIndex.add(entityId);
+		entityIndex.add(entityId.toLowerCase());
```

Comment:
Oh man, is this why this `expand` method never really seem to do anything. Oh my.